larray/core/axis.py

```diff
@@ -2094,39 +2099,81 @@ def combine_axes(self, axes=None, sep='_', wildcard=False, front_if_spread=False
         -------
         AxisCollection
             New AxisCollection with combined axes.
+            
+        Examples
+        --------
+        >>> axes = AxisCollection('a=a0..a2;b=b0..b2')
+        >>> axes
+        AxisCollection([
+            Axis(['a0', 'a1', 'a2'], 'a'),
+            Axis(['b0', 'b1', 'b2'], 'b')
+        ])
+        >>> axes.combine_axes()
+        AxisCollection([
+            Axis(['a0_b0', 'a0_b1', 'a0_b2', 'a1_b0', 'a1_b1', 'a1_b2', 'a2_b0', 'a2_b1', 'a2_b2'], 'a_b')
+        ])
+        >>> axes.combine_axes(sep='/')
```

Comment:
I think there are too many examples in here. I would move some of them to unit tests. But wait until we are sure of the syntax before changing them.

Reply:
normally, very few users will use `AxisCollection.combine_axes`. So I don't think it is really a problem to have many examples.